src/flick/flick/settings.py

```diff
@@ -31,6 +30,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+# turning running with VALIDATE_SOCIAL_TOKEN will check social tokens before registering a user
```

Comment:
what does `turning running` mean?

Reply:
sorry typo with comment. so I add a variable in setting that you can turn on or off for validating social token. It might be annoying to have the check on while development?

Reply:
could you update this comment then!